app/models/label/LabelTable.scala

```diff
@@ -410,6 +423,155 @@ object LabelTable {
     selectQuery(labelId).list.map(label => labelAndTagsToLabelMetadata(label, getTagsFromLabelId(label._1))).head
   }
 
+  /**
+    * This method returns a LabelMetadata object for the validation interface.
+    * Sends relevant information for rendering onto a GSV panorama.
+    * @param labelId  Label ID from query.
+    * @return         LabelMetadata object.
+    */
+  def retrieveSingleLabelForValidation(labelId: Int): LabelValidationMetadata = db.withSession { implicit session =>
+    val selectQuery = Q.query[Int, (Int, String, String, Float, Float, Float, Int, Int, Int, Int)](
```

Comment:
Hmm... I don't think so! I think I was just following what was done for the `LabelMetadata` queries. 

Anyways, this is fixed now! There's one more query that's more complicated that I've been trying to work on `retrieveSingleRandomLabelForValidation` that has been a bit tricky, so that hasn't been updated yet.